### DISCUSSION TITLE
Domain/sub-domain option

### DISCUSSION THREAD

**Discussion by crashzk** [ianlucas/cs2-inventory-simulator#95] [General]
Some previous versions I noticed that I had the option to host the Inventory Simulator, point a new domain to the same project. However, this new domain would have its own link to access it, as well as the skins they choose are saved only for this domain.

Is this still possible? If so, can we expect some documentation, or some simple tutorial on how to do it?

What I would like is to use my website, and with it put other domains for users to access. However, if they access this domain it does not show the skins they have chosen. Each domain has its own collection of skins, without synchronizing with the main domain.

**Comment by ianlucas** [comment]
We had a feature similar to what you described, but I removed it. No current plans to reimplement it, most of the time is more simple to just host an instance instead of trying to handle multiple domains in a single application.

**Reply by crashzk** [reply]
I understand, I thought about the use of machine resources. Installing several instances would use more machine resources, I believe that the method I had before as mentioned would not have this problem.

Anyway, thank you!
